xarray/core/dataarray.py

```diff
@@ -2692,6 +2692,12 @@ def diff(self, dim: Hashable, n: int = 1, label: Hashable = "upper") -> "DataArr
         difference : same type as caller
             The n-th order finite difference of this object.
 
+        .. note::
+
+            `n` matches numpy's behavior and is different from pandas' similarly
```

Comment:
I think "similarly located" is confusing.

How about 

```
The first argument `n` matches numpy's behavior and is different from pandas' first argument named `periods`.
```

Reply:
Yes much better